features/admin/pod.feature

```diff
@@ -102,6 +102,7 @@ Feature: pod related features
   # @case_id OCP-11752
   @admin
   @inactive
+  @aws-ipi
```

Comment:
@jhou1 Perhaps we do not need to add the tag when it already has tag inactive ?

Reply:
@liangxia I think if someone activates the test case and removes the `@inactive` tag from the code base, then the prow tags are out of sync. I suggest we still add the tags anyway in case someone wants to reenable them in the future. We would therefore have less effort keeping things up to date.
WDYT?

Reply:
I also made a mistake in the test case query, inactive test cases should not be filtered in my query anyway, fixing everything now...

Reply:
Make sense now.    I think the mistake makes most of the test cases added with tag aws-ipi also have inactive tag, that's why the concern comes from.